### PR TITLE
Demo server

### PR DESCRIPTION
**Pull request by sabel83**

### CODE REVIEW

**Comment by r0mai** [comment]
:+1:

File: tools/demo_server/config/metashell.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "metashell",
+    "git": "https://github.com/sabel83/metashell.git",
+    "build": [
+      [
+        "/bin/bash",
+        "-c",
+        "git log -1 --pretty=\"%h %B\" | head -1 | sed 's/\\\\/\\\\\\\\/g' | sed 's/\"/\\\\\"/g' | sed 's/^/\"/' | sed 's/$/\\\\n This demo is supported by the Eotvos Lorand University, Faculty of Informatics\"/' > lib/core/version_desc.hpp"
```
Comment (r0mai) [inline tools/demo_server/config/metashell.json]: This probably shouldn't be hardcoded, but exposed as an environment variable.

Comment (r0mai) [inline tools/demo_server/config/metashell.json]: Or, at least it should have a different default value. what do you think?

But we could say that this is a sample config file which is also used as is by the official demo, but then we should document it in the docs.

Comment (sabel83) [inline tools/demo_server/config/metashell.json]: We could move the "supporter" text into a local file (outside of the git repository) and make the build pick it up from that.

Comment (r0mai) [inline tools/demo_server/config/metashell.json]: That's a good idea, yes.

**Pull request merged**